sympy/stats/tests/test_continuous_rv.py

```diff
@@ -430,6 +430,7 @@ def test_gumbel():
     x = Symbol("x")
     X = Gumbel("x", beta, mu)
     assert simplify(density(X)(x)) == exp((beta*exp((mu - x)/beta) + mu - x)/beta)/beta
+    assert cdf(X)(x) == exp(-exp(-mu + x)/beta)
```

Comment:
This should return `exp(-exp((mu - x)/beta))`.

Reply:
I have made the corrections. Please check @sidhantnagpal .